src/Symfony/Bundle/FrameworkBundle/DependencyInjection/FrameworkExtension.php

```diff
@@ -1180,6 +1181,18 @@ private function registerRouterConfiguration(array $config, ContainerBuilder $co
                 new Reference('file_locator'),
                 new Reference('routing.loader.annotation'),
             ]);
+
+
+        $container->register('routing.loader.psr4', Psr4DirectoryLoader::class)
```

Comment:
ah sorry, I wasn't clear enough: then this should be moved to routing.php

Reply:
let's move them all? It makes things easier to discover when they're all in the same place

Reply:
Done.